atk/src/auto/text.rs

```diff
@@ -45,22 +45,22 @@ pub trait TextExt: 'static {
     ) -> Vec<TextRange>;
 
     #[doc(alias = "atk_text_get_caret_offset")]
-    fn get_caret_offset(&self) -> i32;
+    fn caret_offset(&self) -> i32;
 
     #[doc(alias = "atk_text_get_character_at_offset")]
     fn get_character_at_offset(&self, offset: i32) -> char;
 
     #[doc(alias = "atk_text_get_character_count")]
-    fn get_character_count(&self) -> i32;
+    fn character_count(&self) -> i32;
 
     #[doc(alias = "atk_text_get_character_extents")]
     fn get_character_extents(&self, offset: i32, coords: CoordType) -> (i32, i32, i32, i32);
```

Comment:
Any reason why those were not renamed?

Reply:
See https://github.com/fengalin/fix-getters#get-functions-selection
This is open to discussion of course.